python/cudf/dataframe/series.py

```diff
@@ -1037,6 +1037,32 @@ def hash_values(self):
 
         return Series(numerical.column_hash_values(self._column))
 
+    def hash_encode(self, stop, use_name=False):
```

Comment:
Since we're exposing a new function to the external API here, is there an API in Pandas / Numpy / SKLearn we could follow to give users familiarity?

Reply:
SKLearn has a Transformer: https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.FeatureHasher.html

The other's don't have anything similar to the best of my knowledge.

Reply:
Hmm, that API wouldn't translate nicely into a member function of Series. Let's merge this as is and see if there's any user feedback.